source/includes/_catalog.md

```diff
@@ -169,9 +170,9 @@ KillBillClient::Model::Catalog.get_tenant_catalog_xml(requested_date,
 ```
 
 ```python
-catalogApi = killbill.api.CatalogApi()
+catalogApi = killbill.CatalogApi()
 
-catalogApi.get_catalog_xml(api_key, api_secret)
+catalogXml = catalogApi.get_catalog_xml()
```

Comment:
would suggest renaming this and other variables to camel case (like `catalog_xml`)